### ISSUE TITLE
Read simulation CL arguments from file

### ISSUE THREAD

**Issue by aturfah** [aturfah/cmplxsys530-final#22]
Proportions are kind of annoying to do from CL (for all the possible cases), option to read from file would be nice (and also let me run different versions of the simulations instead of having to remember them).

**Comment by aturfah** [comment]
Lines should be formatted as follows (spacing optional):
```
# Comment lines are ignored
argument1 | value
argument2 | value
...
```

Ex:
```
# MultiTurn RPS Simulation w/ 20 players for 5000 rounds
np | 20
ng | 10000
game_choice | 3
proportions | 0.2 0.2 0.2 0.3 0.1
```

**Comment by aturfah** [comment]
Addressed in #23
